From f11dec4f94f9daafec2e7f2ca59a554c97dd8e1f Mon Sep 17 00:00:00 2001
From: Will Dietz <w@wdtz.org>
Date: Thu, 9 May 2024 08:20:10 -0500
Subject: [PATCH] [PPC][lit] Catch and ignore any exception running 'git'.

specifically, OSError for difficulties executing
(e.g., no such file or directory).
---
 llvm/test/CodeGen/PowerPC/lit.local.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/llvm/test/CodeGen/PowerPC/lit.local.cfg b/llvm/test/CodeGen/PowerPC/lit.local.cfg
index 4cc802afef4a..0cebf1b7b22a 100644
--- a/llvm/test/CodeGen/PowerPC/lit.local.cfg
+++ b/llvm/test/CodeGen/PowerPC/lit.local.cfg
@@ -9,7 +9,7 @@ def get_revision(repo_path):
     cmd = ['git', '-C', repo_path, 'rev-parse', 'HEAD']
     try:
         return subprocess.run(cmd, stdout=subprocess.PIPE, check=True).stdout.decode()
-    except subprocess.CalledProcessError:
+    except:
         print("An error occurred retrieving the git revision.")
         return None
 
-- 
2.43.2

